Migration.md

```diff
@@ -5,6 +5,32 @@ Deprecated code produces compile-time warnings. These warning serve as
 notification to users that their code should be upgraded. The next major
 release will remove the deprecated code.
 
+## Gazebo Math 8.X to 9.X
+
+### Breaking Changes
+
+1. **graph/Graph.hh**
+    + The functions `Graph::AddVertex()` and `Graph::VertexFromId()` might
+      return NullVertex() instead of NullVertex.
+      E.g.: https://github.com/gazebosim/gz-math/pull/606/files#diff-0c0220a7e72be70337975433eeddc3f5e072ade5cd80dfb1ac03da233c39c983L153-R153
+
+    + The functions `Graph::AddEdge()`, `Graph::LinkEdge()`,
+      `Graph::EdgeFromVertices()` and `Graph::EdgeFromId()` functions might
+      return NullEdge() instead of NullEdge.
+      E.g.: https://github.com/gazebosim/gz-math/pull/606/files#diff-0c0220a7e72be70337975433eeddc3f5e072ade5cd80dfb1ac03da233c39c983L222-R222
```

Comment:
I don't think these are breaking change in `Graph.hh`. I had originall commented on `Graph.hh` because that is where I noticed that our API was changing due to the previously removed type, but for this PR, I think the only needed change to the Migration guide is the deprecations below

Reply:
You're right, we're not changing the function signature so it should be fine.